src/blocks/shared/utils/api.ts

```diff
@@ -0,0 +1,34 @@
+export interface AnalyticsApiQueryParams extends AnalyticsApiOptionalQueryParams {
```

Comment:
I don't see us using this somewhere. Do we plan on using this in the future?

Reply:
In future maybe ..... I think we should have a common definition of API interfaces in the codebase so in other places where we are defining local interfaces we should use these common interfaces.

It is not used but I have kept it here just to match it with the API specs i.e. just the interface is enough to get full idea of API params.